moveit_ros/moveit_servo/src/collision_check.cpp

```diff
@@ -49,21 +49,21 @@ constexpr size_t ROS_LOG_THROTTLE_PERIOD = 30 * 1000;  // Milliseconds to thrott
 namespace moveit_servo
 {
 // Constructor for the class that handles collision checking
-CollisionCheck::CollisionCheck(const rclcpp::Node::SharedPtr& node, const ServoParameters::SharedConstPtr& parameters,
+CollisionCheck::CollisionCheck(const rclcpp::Node::SharedPtr& node, const servo::Params& servo_params,
                                const planning_scene_monitor::PlanningSceneMonitorPtr& planning_scene_monitor)
   : node_(node)
-  , parameters_(parameters)
+  , servo_params_(servo_params)
```

Comment:
It's good that you're creating a copy of these parameters so we don't have to worry about thread safety.